gatsby/content/projects/2018-12-27-fluffychat.mdx

```diff
@@ -4,40 +4,43 @@ title: FluffyChat
 categories: 
  - client
 thumbnail: https://christianpauly.gitlab.io/fluffychat-website/assets/images/screenshot.png
-description: Open, nonprofit and cute client for Ubuntu Touch
+description: Chat with your friends
```

Comment:
As all client projects are supposedly chat applications, it would be great to see what makes FluffyChat different.
I see the change of the description as a degression, though it's great to see more platforms being supported. 👍

Reply:
Okay, I have updated a better description